services/docker-compose.yml

```diff
@@ -264,30 +264,6 @@ services:
     networks:
       - default
 
```

Comment:
you might want to keep this config in a separate docker-compose config file under scripts for the future?